extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -637,7 +637,7 @@ public IngestTester testIngestQuery()
     return new IngestTester();
   }
 
-  private ObjectMapper setupObjectMapper(Injector injector)
+  public static ObjectMapper setupObjectMapper(Injector injector)
```

Comment:
This method is deprecated. (I should add a `@Deprecated` annotation.). The preferred solution is to register the module that contains the item here, or create a "stub" module with the item needed, and use the methods to register Druid modules.